index.js

```diff
@@ -724,6 +724,11 @@
     return x instanceof Left && R.eqProps('value', x, this);
   });
 
+  //  Left#extend :: Either a b ~> (b -> b) -> Either a b
+  Left.prototype.extend = def('Left#extend', [Function], function(f) {
+    return this;
+  });
```

Comment:
There's a `self` function defined near the top of the file as `function() { return this; }`. Let's use that here. :)